File: baekjoon/bronze/level1/test_2693/Main.java
package bronze.level1.test_2693;

import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = Integer.parseInt(sc.nextLine());

        for (int i = 0; i < n; i++) {
            int[] arr = Arrays.stream(sc.nextLine().split(" ")).mapToInt(Integer::parseInt).toArray();
            Arrays.sort(arr);
            System.out.println(arr[7]);
        }
    }
}

File: baekjoon/bronze/level1/test_2693/kotlin/Main.kt
package baekjoon.bronze.level1.test_2693.kotlin

fun main() {
    val n = readln().toInt()

    for (i in 1..n) {
        var arr = readln().split(" ").map { it.toInt() }.sorted()
        println(arr[7])
    }
}
